src/components/GraphSidebar/ElementContainer.svelte

```diff
@@ -3,12 +3,16 @@
   import type { IconDefinition } from "@fortawesome/free-solid-svg-icons";
   import { createEventDispatcher } from "svelte";
 
+  import type { IconOptionsDefinition } from "./types";
+
   export let label: string;
   export let icon: IconDefinition;
+  export let iconOptions: Array<IconOptionsDefinition> | null = null;
```

Comment:
Maybe it's better to set it to
`export let iconOptions: Array<IconOptionsDefinition> | undefined;`

Reply:
done